sensor.py

```diff
@@ -34,9 +48,10 @@ async def async_setup_platform(hass, config, async_add_entities,
 
 class HSSensor(Entity):
     """Representation of a HomeSeer sensor-type device."""
-    def __init__(self, device, connection):
+    def __init__(self, device, connection, units=None):
```

Comment:
See my comments below, this additional keyword parameter will not be required.

Reply:
This will be true if we can figure out a way to get the unit system from HS but I can't find any way to do that using the API.

Reply:
Looking at the API I see this request: 

`/JSON?request=getsetting&setting=SETTING_NAME`

The documentation gives the example:

```
 Returns the value for a specific settting. For example, the setting for the name of location 1 is called "gLocLabel". To get the name of this label use:

/JSON?request=getsetting&setting=gLocLabel

The return might be:

{
"Value":"Room"
}
```

Possibly the localization/temperature unit setting could be obtained this way?

Reply:
I don't see anything in settings.ini that indicates the temperature scale setting.   Taking a look around some other files.

Reply:
I did some searching on the HS forums and this has been an ongoing problem for various setups.  Many of the devices have their own scale setting as a part of their protocol while others rely on the HS setting.  For example, the TEMP05 1-wire logger has a command to set the scale for its readings and also an option to append "F" or "C" to the end of the readings.  When sending these readings to HA via MQTT using device_string instead of device_value, it's easy to determine the units based on the char at the end of the string. 

I'm not even sure a configuration key would solve this since readings could be in either scale depending upon how the device or devices were configured on the HS side.  I discovered this with light sensors as well.  Some of my sensors report in percentage from 0-100% while others report in Lux from 0 - ~2500.  

I'm wondering if it would be better not to handle this in the component at all and set the units using entity customization. I'm new to HA so still learning.

Reply:
I think you might be right here - given there's no consistency, we should just report the value without units or icon modifications, and let the user customize. No real need for a HSTemperature class in that case either.

Reply:
Another question - since this component only handles ZWave devices (and I would want to keep it that way) is temp reporting part of the ZWave spec at all? I would think that maybe all ZWave devices report in Fahrenheit?

Reply:
Yes, It's a part of the HRV Status Command Class.  Section 4.45, specifically 4.45.2 HRV Status Report Command.